src/bsfs.c

```diff
@@ -675,6 +687,10 @@ static int dealloc_clusters(bs_open_level_t level,
   return 0;
 }
 
+size_t get_required_cluster_count(off_t file_size) {
```

Comment:
This function should be documented.
After all, this segment of code was documented before it became a function.

Reply:
Since the refactor, this function is practically self-documenting, but I've added a short comment explaining the calculation in a6f43a0.